web_graph_sort/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Graph Sort
+==============
+
+This module allows to sort pivot tables.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to any pivot table. Click on a column header. The table will be sorted by that column.
+
```

Comment:
Maybe a user doesn't know what is a pivot table. You could clarify a lot saying Go To Reporting -> ....